docs/workflows/build-containerfile.rst

```diff
@@ -65,3 +65,9 @@ Build an OCI image
 
 .. literalinclude:: ../_scripts/build_containerfile.sh
    :language: bash
+
+
+.. warning::
+    Users lacking read access to the ``artifacts`` endpoint may encounter restricted functionality
```

Comment:
did you just say in other words that this is restricted to admin-only users? :)

Reply:
Yes. Is it understandable? I could reword it to contain the "non-staff users" term.